### DISCUSSION TITLE
Welcome to ha-chore-helper Discussions!

### DISCUSSION THREAD

**Discussion by ReplicaObscura** [VolantisDev/ha-chore-helper#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vince159** [comment]
Hi, I'm using this integration and I'm very happy with it. I asked you how I could build an automation that when chore_helper.complete = 0 then sends me a notification on the different devices. Thank you.
